employee-rostering-benchmark/src/main/resources/employeeRosteringBenchmarkConfig.xml

```diff
@@ -0,0 +1,171 @@
+    <solver>
+      <localSearch>
+        <unionMoveSelector>
+          <changeMoveSelector/>
+          <swapMoveSelector/>
+          <pillarChangeMoveSelector/>
+          <pillarSwapMoveSelector/>
+        </unionMoveSelector>
+      </localSearch>
+    </solver>
+  </solverBenchmark>
+
+  <solverBenchmark>
+    <name>Move Selector and Sequential Pillar Move Selector</name>
+    <solver>
+      <localSearch>
+        <unionMoveSelector>
+          <changeMoveSelector/>
+          <swapMoveSelector/>
+          <pillarChangeMoveSelector>
+            <subPillarType>SEQUENCE</subPillarType>
+            <subPillarSequenceComparatorClass>org.optaweb.employeerostering.solver.ShiftComparator</subPillarSequenceComparatorClass>
+            <pillarSelector>
+              <minimumSubPillarSize>1</minimumSubPillarSize>
+              <maximumSubPillarSize>10</maximumSubPillarSize>
+            </pillarSelector>
+          </pillarChangeMoveSelector>
+          <pillarSwapMoveSelector>
+            <subPillarType>SEQUENCE</subPillarType>
+            <subPillarSequenceComparatorClass>org.optaweb.employeerostering.solver.ShiftComparator</subPillarSequenceComparatorClass>
+            <pillarSelector>
+              <minimumSubPillarSize>1</minimumSubPillarSize>
+              <maximumSubPillarSize>10</maximumSubPillarSize>
+            </pillarSelector>
+          </pillarSwapMoveSelector>
+        </unionMoveSelector>
+      </localSearch>
+    </solver>
+  </solverBenchmark>
+
+  <solverBenchmark>
+    <name>Long Tabu Search</name>
+    <solver>
+      <localSearch>
+        <acceptor>
+          <entityTabuSize>11</entityTabuSize>
+        </acceptor>
+        <forager>
+          <acceptedCountLimit>1100</acceptedCountLimit>
+        </forager>
+      </localSearch>
+    </solver>
+  </solverBenchmark>
+
+  <solverBenchmark>
+    <name>Medium Tabu Search</name>
+    <solver>
+      <localSearch>
+        <acceptor>
+          <entityTabuSize>7</entityTabuSize>
+        </acceptor>
+        <forager>
+          <acceptedCountLimit>800</acceptedCountLimit>
+        </forager>
+      </localSearch>
+    </solver>
+  </solverBenchmark>
+
+  <solverBenchmark>
+    <name>Simulated Annealing</name>
+    <solver>
+      <localSearch>
+        <acceptor>
+          <simulatedAnnealingStartingTemperature>0hard/0medium/8soft</simulatedAnnealingStartingTemperature>
+          <entityTabuSize>5</entityTabuSize>
+        </acceptor>
+        <forager>
+          <acceptedCountLimit>4</acceptedCountLimit>
+        </forager>
+      </localSearch>
+    </solver>
+  </solverBenchmark>
+
+  <solverBenchmark>
+    <name>Late Acceptance</name>
+    <solver>
+      <localSearch>
+        <acceptor>
+          <lateAcceptanceSize>500</lateAcceptanceSize>
+        </acceptor>
+        <forager>
+          <acceptedCountLimit>1</acceptedCountLimit>
+        </forager>
+      </localSearch>
+    </solver>
+  </solverBenchmark>
```

Comment:
I don't like the fact that we're mixing benchmarks for selectors with benchmarks for acceptors/foragers. If my memory serves me correctly, in `optaplanner-examples` we solve that by having different benchmark config files.

In this case, I think that doesn't help either. We want to show benchmarking capabilities, but not necessarily show everything that could be benchmarked. That's what the documentation is for.

In my opinion, if the config includes just the selector benchmarks, it will be better. It will show benchmarking is possible, but not overwhelm users.

Reply:
Fair enough. Removing these will also decrease testing time.